municipal_finance/resources.py

```diff
@@ -193,7 +193,7 @@ class Meta:
 class AgedCreditorItemsV1Resource(resources.ModelResource):
     class Meta:
         model = AgedCreditorItemsV1
-        import_id_fields = ("code",)
+        import_id_fields = ("code","id")
```

Comment:
Just checking but in `aged_creditor.json` it says `"join_column": ["item_id", "id"],` while here it says `import_id_fields = ("code","id")`. Is `item_id` and `code` right?